src/command_args.rs

```diff
@@ -636,6 +670,7 @@ fn test_invalid_private_key() {
         matches,
         config: Some(ConfigToml {
             signer: Some(SignerToml {
+                to_address: Some("mbaffffff".to_string()),
```

Comment:
This test spec is a test for `privatekey: Some ("aabbccdd" .to_string ())`, so couldn't verify the error of to_address?

Reply:
Yes, my change's intention is just path the compile. but so that, it should be `None`. I'll change it.

Reply:
In addition to it, test for address format check also need.

Reply:
Yes, i also add it.

Reply:
I've added 2 commits for it.